src/modules/FunModule.cpp

```diff
@@ -66,6 +67,46 @@ FunModule::FunModule(UmikoBot* client) : Module("funutil", true), m_memeChannel(
 
 		});
 
+	RegisterCommand(Commands::FUN_ROLL, "roll", [this](Discord::Client& client, const Discord::Message& message, const Discord::Channel& channel)
+	{
+
+		QStringList args = message.content().split(' ');
+		GuildSetting* setting = &GuildSettings::GetGuildSetting(channel.guildId());
+		QString prefix = setting->prefix;
+
+		int max;
+		int min;
+
+		if (args.size() == 1) 
+		{
+			client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+			return;
+		}
+
+		if (args.size() == 2)
+		{
+			min = 1;
+		}
+		else
+		{
+			min = args.at(2).toDouble();
+		}
+		min = args.at(1).toDouble();
```

Comment:
Think you forgot to swap these out. surely `min` is the first argument in the command? 😛

Reply:
lol yeah